protostar/commands/test/test_command.py

```diff
@@ -108,6 +108,12 @@ def arguments(self) -> List[Command.Argument]:
                 type="int",
                 description="Set a seed to use for all fuzz tests.",
             ),
+            Command.Argument(
+                name="report-slowest-tests",
+                type="int",
+                description="Print slowest tests at the end.",
+                default=0,
+            ),
```

Comment:
Why is this a configurable argument? IMHO people will not use this feature not even know that it exists, because it's disabled by default